src/SynoliaSyliusGDPRPlugin.php

```diff
@@ -7,15 +7,19 @@
 use Sylius\Bundle\CoreBundle\Application\SyliusPluginTrait;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Synolia\SyliusGDPRPlugin\DependencyInjection\CompilerPass\RegisterAdvancedActionsFormDataProcessorsPass;
 use Synolia\SyliusGDPRPlugin\DependencyInjection\CompilerPass\RegisterAnonymizationLoader;
 
 final class SynoliaSyliusGDPRPlugin extends Bundle
 {
     use SyliusPluginTrait;
 
+    public const VERSION = '1.1.0';
```

Comment:
Is it possible to get it automatically? Because we can forget toi update this. And value can't be 1.1.0 but 1.2.0 ou 2.0.0 following the content of the next tag.

Reply:
I don't know how i can do this :/

Reply:
Ok, so I'd rather we didn't display the version than display the wrong version.
Can you remove it ?